pkg/etcdcli/etcdcli.go

```diff
@@ -61,8 +63,16 @@ func (g *etcdClientGetter) getEtcdClient() (*clientv3.Client, error) {
 	}
 	for _, addr := range hostEtcd.Subsets[0].Addresses {
 		if addr.Hostname == "etcd-bootstrap" {
-			// etcd-bootstrap has a valid IP in host-etcd
-			etcdEndpoints = append(etcdEndpoints, fmt.Sprintf("https://%s:2379", addr.IP))
+			// etcd-bootstrap has a valid IP in host-etcd-2
+			hostEtcd2, err := g.endpointsLister.Endpoints(operatorclient.TargetNamespace).Get("host-etcd-2")
+			if err != nil {
+				return nil, err
+			}
+			ip, ok := hostEtcd2.Annotations[BootstrapIPAnnotationKey]
+			if !ok || ip == "" {
+				return nil, fmt.Errorf("service/host-etcd-2 annotation %s missing IP address", BootstrapIPAnnotationKey)
+			}
+			etcdEndpoints = append(etcdEndpoints, fmt.Sprintf("https://%s:2379", ip))
```

Comment:
need escape if ipv6.  Look at my ipv6 PR and dnshelpers

Reply:
we should check it to be sure but I escape the annotation value if IPv6

/home/remote/sbatsche/projects/openshift/installer/bin/openshift-install

Reply:
Don't escape the annotation.